website/docs/d/pi_workspace.html.markdown

```diff
@@ -7,22 +7,26 @@ description: |-
 ---
```

Comment:
what's the consensus on 
````
page_title: "IBM: pi_workspace" 
```
vs 
```
page_title: "IBM: ibm_pi_workspace"
```

?

Reply:
None -- We probably need to ask Harini about this later.